src/components/appheadernavigation.main.jsx

```diff
@@ -137,18 +136,6 @@ class AppHeaderNavigationMain extends React.Component {
             Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
           },
         }))
-      .then((res) => {
```

Comment:
@petro97 I'm doing some clean up here since we no longer need this code if it's in Cortex.js